api/src/main/java/marquez/db/DatasetFieldDao.java

```diff
@@ -129,7 +129,7 @@ void deleteDatasetVersionFieldTag(
 
   @SqlUpdate(
       "INSERT INTO dataset_fields_tag_mapping (dataset_field_uuid, tag_uuid, tagged_at) "
-          + "VALUES (:rowUuid, :tagUuid, :taggedAt)")
```

Comment:
Can you add a Java language test for this?

Reply:
Cool will add an additional insert which should result in no change.

Reply:
Thanks